pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -372,6 +457,16 @@ def newpkgcb(pkgId, name, arch):
                                 deferred_download=self.deferred_download
                             )
                             dc = DeclarativeContent(content=package, d_artifacts=[da])
+                            dc.extra_data = defaultdict(list)
+
+                            # find if a package relates to a modulemd
+                            if dc.content.nevra in nevra_to_module.keys():
+                                dc.content.is_modular = True
+                                for dc_modulemd in nevra_to_module[dc.content.nevra]:
```

Comment:
This loop and the one below iterates the same objects. Can we combine them into one loop?

Reply:
we can, good point here